Makefile

```diff
@@ -57,19 +57,19 @@ install:
 TAR_FLAGS=--transform 's/.*\///g' --owner=0 --group=0
 
 artifacts: clean
-	GOOS=linux GOARCH=amd64       make -C $(CURDIR)  binaries
+	GOOS=$(GOOS) GOARCH=amd64       make -C $(CURDIR)  binaries
```

Comment:
This produces `nerdctl-$(VERSION_TRIMMED)-linux-amd64.tar.gz`, so `GOOS` has to be hard-coded to `linux` here.

Instead, we should add `GOOS=windows` that produces  `nerdctl-$(VERSION_TRIMMED)-windows-amd64.zip`